specification.markdown

```diff
@@ -359,7 +359,7 @@ The path defines the allowed storage locations of the EBML Element within an EBM
 The path attribute is REQUIRED.
 
 ```
-EBMLFullPath             = EBMLElementOccurrence "(" EBMLMasterPath ")"
```

Comment:
@robUx4, here I just removed the min white space needed to adhere to the line length needs of the output, but perhaps it would be better to rename `VariableParentOccurrence` to something shorter and shift everything to the left

Reply:
I'll rename *Variable* to *Global*